scss/base/_global.scss

```diff
@@ -0,0 +1,54 @@
+/* ---------------------------------------------------- */
+/* ----- GLOBAL STYLES ----- */
+/* ---------------------------------------------------- */
+
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+*::before, *::after {
+	box-sizing: border-box;
+}
+
+html, body {
+	width: 100%;
+	height: 100%;
+	overflow-x: hidden;
+}
+
+body {
+	font-family: $font-base;
+	font-weight: 400;
+	line-height: 1.5;
+}
+
+img {
+	max-width: 100%;
+	display: block;
+}
+
+li {
+	list-style: none;
+}
+
+h1 {
+	font-weight: 400;
+}
+
+/* --- CLEARFIX --- */
+.clearfix {
+	@include clearfix;
```

Comment:
Sama to napisałaś czy ładujesz jakiś preset? Bądź ostroszna z ładowanie zewnętrzynch styli. Często to tylko niepotrzebny "szum" w kodzie.